packages/ubus-lime-metrics/files/etc/uci-defaults/99-save-last-known-path-to-internet

```diff
@@ -1,5 +1,4 @@
 #!/bin/sh
 echo "adding cron to save last known path to the Internet"
-sed "\| \* \* \* .*last_internet_path.*|d" -i /etc/crontabs/root
-echo "*/30 * * * * /usr/sbin/internet_path > /tmp/last_internet; if [ -s /tmp/last_internet ] && [\ "$(md5sum /tmp/last_internet | awk '{ print $1 }')\" != \"$(md5sum /etc/last_internet |  awk '{ print $1 }
-')\" ]; then cp /tmp/last_internet /etc/last_internet; fi" >>  /etc/crontabs/root
+sed "\| \* \* \* .*internet_path*|d" -i /etc/crontabs/root
+echo "*/30 * * * * /usr/sbin/internet_path save" >>  /etc/crontabs/root
```

Comment:
Don't you prefere the unique_append idiom, lime in other packages?

Reply:
Yeah, this package is older and I just fixed the script. I can adapt it to the unique_append style

Reply:
better use the unique_append (is this the exact name?) idiom then ;)